jbpm-wb-process-runtime/jbpm-wb-process-runtime-backend/src/main/java/org/jbpm/workbench/pr/backend/server/RemoteProcessVariablesServiceImpl.java

```diff
@@ -80,16 +91,19 @@
         }
 
         Map<String, String> properties = new HashMap<String, String>();
+        if (isContainerAvailable(serverTemplateId,
```

Comment:
@nmirasc I think you can still use the query service to get the variables definition.
Something like: 
`queryServicesClient.findProcessesById(processId).get(0).getProcessVariables()`
That way you could skip the container check?

Reply:
@cristianonicolai I have one doubt about this replacement. It returns the first process with that id but it doesn't ensure that is the one related with the process instance. It could be one with the same processId from other container.

Reply:
@nmirasch there is another method, _findProcessByContainerIdProcessId_ that might be the one to use then.